project_2/include/express_pass.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "packet.hpp"
+
+#include <random>
+
+class ExpressPass {
+public:
+    const static std::uint64_t min_packet_size = 84 * 8;
```

Comment:
In the paper they used this value as a minimum size of credit packet, I think it depends on the implementation, so in the future it will change to something more relevant for our case